src/backend/helpers/pizza.js

```diff
@@ -0,0 +1,57 @@
+const { Op, Sequelize } = require("sequelize");
+
+const buildLikeFilter = (value, table, col) => {
+    return Sequelize.where(
+        Sequelize.fn("LOWER", Sequelize.col(`${table}.${col}`)),
+        "LIKE",
+        "%" + value.toLowerCase() + "%"
+    );
+};
+
+const buildValorFilter = (query, value) => {
+    const filter = {};
+
+    if (query === "valor_maior_que") filter[Op.gte] = value;
+    else if (query === "valor_menor_que") filter[Op.lte] = value;
+    else if (query === "valor") filter[Op.eq] = value;
+
+    return filter;
+};
+
+const buildWhere = (query) => {
+    const {
+        created_at_start,
+        created_at_end,
+        updated_at_start,
+        updated_at_end,
+        ...others
+    } = query;
+
+    const where = Object.entries(others).reduce((query, [key, value]) => {
+        query[key] =
+            typeof value == "number"
```

Comment:
will this work? as the value comes from the query string, won't it be of type "string" for every case ?